build.gradle

```diff
@@ -11,6 +11,26 @@
 
 import java.nio.file.Files
 import org.gradle.api.internal.tasks.testing.junitplatform.JUnitPlatformTestFramework
+import org.opensearch.gradle.test.RestIntegTestTask
+import org.opensearch.gradle.testclusters.ExtensionsProperties
+
+buildscript {
+    ext {
+        opensearch_version = System.getProperty("opensearch.version", "3.0.0-SNAPSHOT")
+        isSnapshot = "true" == System.getProperty("build.snapshot", "true")
+        buildVersionQualifier = System.getProperty("build.version_qualifier", "")
```

Comment:
Just curious, what is the significance of  buildVersionQualifier and why "" has been set here?

Reply:
It's an identifier that can potentially be added on to a version constant.  Here, it isn't inherently set to "".  That is the default value for the variable in case the System property can't be found or is empty.